libr/core/linux_heap_glibc.c

```diff
@@ -21,6 +21,37 @@
 #define GHT_MAX UT64_MAX
 #endif
 
+static GHT GH(get_va_symbol)(const char *path, const char *symname) {
+	RListIter *iter;
+	RBinSymbol *s;
+	RBin *bin = r_bin_new ();
+	RIO *io = r_io_new ();
+	RList *syms = NULL;
+	GHT vaddr = 0LL;
+	r_io_bind (io, &bin->iob);
+
+	if (!bin) {
+		return 0;
```

Comment:
i think GHT_MAX should be returned in case of error. i know address 0 shouldnt be valid place. but -1 should never be a valid address to place any symbol. and it makes the code more consistent to read